server/routerlicious/packages/services-client/src/test/array.spec.ts

```diff
@@ -68,3 +73,49 @@ describe("convertToRanges", () => {
 		assert.strictEqual(JSON.stringify(ranges), JSON.stringify([[0, 999]]));
 	});
 });
+
+describe("mergeArrays", () => {
```

Comment:
Can you add a test case for an array with some duplicate values?

Reply:
Added